tests/app/commands/test_populate_redis.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+from freezegun import freeze_time
+import pytest
+
+from app.commands import populate_redis_template_usage
+
+from tests.conftest import set_config
+from tests.app.db import create_notification, create_template, create_service
+
+
+def test_populate_redis_template_usage_does_nothing_if_redis_disabled(mocker, notify_api, sample_service):
+    mock_redis = mocker.patch('app.commands.redis_store')
+    with set_config(notify_api, 'REDIS_ENABLED', False):
+        with pytest.raises(AssertionError):
+            populate_redis_template_usage.callback.__wrapped__(sample_service.id, datetime.utcnow())
+
+    assert not mock_redis.called
+
+
+def test_populate_redis_template_usage_does_nothing_if_no_data(mocker, notify_api, sample_service):
+    mock_redis = mocker.patch('app.commands.redis_store')
+    with set_config(notify_api, 'REDIS_ENABLED', True):
+        populate_redis_template_usage.callback.__wrapped__(sample_service.id, datetime.utcnow())
+
+    assert not mock_redis.called
```

Comment:
Test its called with the right parameters because this could be called is there is data?

Reply:
https://www.songkick.com/concerts/33124154-65daysofstatic-at-queen-elizabeth-hall-southbank-centre?utm_source=8123&utm_medium=partner&utm_content=d16deb28b60332f7aed0a29297a35950&utm_campaign=entity

Reply:
no - this test has no data in the DB, so redis isn't being called.

That said, I do think this isn't the right call. i should make sure none of redis's child objects (the actual set fns) are being called.